server/test/controllers/RedirectControllerTest.java

```diff
@@ -1,377 +0,0 @@
-import static org.mockito.Mockito.when;
-import static play.api.test.CSRFTokenHelper.addCSRFToken;
-import static play.mvc.Http.Status.OK;
-import static play.test.Helpers.contentAsString;
-import static support.CfTestHelpers.requestBuilderWithSettings;
-
-import auth.ProfileUtils;
-import com.google.common.collect.ImmutableList;
-import controllers.applicant.RedirectController;
-import java.util.List;
-import java.util.Locale;
-import models.Applicant;
-import models.Application;
-import models.LifecycleStage;
-import models.Question;
-import org.junit.Before;
-import org.junit.Test;
-import org.mockito.Mockito;
-import org.pac4j.core.client.Client;
-import org.pac4j.core.client.Clients;
-import org.pac4j.core.client.direct.AnonymousClient;
-import org.pac4j.core.client.finder.ClientFinder;
-import org.pac4j.core.config.Config;
-import org.pac4j.core.context.WebContext;
-import org.pac4j.core.engine.DefaultSecurityLogic;
-import play.i18n.Lang;
-import play.i18n.Langs;
-import play.i18n.MessagesApi;
-import play.libs.concurrent.HttpExecutionContext;
-import play.mvc.Result;
-import repository.UserRepository;
-import repository.VersionRepository;
-import services.Path;
-import services.applicant.ApplicantData;
-import services.applicant.ApplicantService;
-import services.applicant.question.Scalar;
-import services.program.EligibilityDefinition;
-import services.program.ProgramDefinition;
-import services.program.ProgramService;
-import services.program.predicate.LeafOperationExpressionNode;
-import services.program.predicate.Operator;
-import services.program.predicate.PredicateAction;
-import services.program.predicate.PredicateDefinition;
-import services.program.predicate.PredicateExpressionNode;
-import services.program.predicate.PredicateValue;
-import support.ProgramBuilder;
-import support.QuestionAnswerer;
-import views.applicant.ApplicantCommonIntakeUpsellCreateAccountView;
-import views.applicant.ApplicantUpsellCreateAccountView;
-
-public class RedirectControllerTest extends WithMockedProfiles {
-
-  @Before
-  public void setUp() {
-    resetDatabase();
-    // Get the config, and hack it so that all requests appear authorized.
-    Config config = instanceOf(Config.class);
-    AnonymousClient client = AnonymousClient.INSTANCE;
-    config.setClients(new Clients(client));
-
-    // The SecurityLogic wants to use some smarts to figure out which client to use, but
-    // those smarts are never going to pick this client (since none of the endpoints are
-    // configured to use it), so we implement an anonymous client finder which always returns
-    // our client.
-    DefaultSecurityLogic securityLogic = new DefaultSecurityLogic();
-    securityLogic.setClientFinder(
-        new ClientFinder() {
-          @Override
-          public List<Client> find(Clients clients, WebContext context, String clientNames) {
-            return ImmutableList.of(client);
-          }
-        });
-    config.setSecurityLogic(securityLogic);
-  }
```

Comment:
In my testing the config logic above isn't needed to get the tests to pass anymore - discovered this while separating out the test files. I think removing this hacky code is warranted. I'm guessing whatever necessitated it in the first place was obsoleted by some kind of Play upgrade.

Reply:
Nice, yeah I think removing this makes sense.